pkg/dnshelper/dnshelper.go

```diff
@@ -0,0 +1,46 @@
+package dnshelper
+
+import (
+	"fmt"
+	"html/template"
+
+	"github.com/massalabs/station/pkg/convert"
+	"github.com/massalabs/station/pkg/node"
+)
+
+const (
+	metaKey = "META"
+
+	// Indexes of data in website name key.
+	indexOfWebsiteAddress     = 0 // Index of website Address in the dnsValue array
+	indexOfWebsiteDescription = 2 // Index of website Description in the dnsValue array
+)
+
+// Helper function to fetch website values from DNS entry.
+func GetWebsiteValues(dnsValue []byte) (string, string, error) {
+	websiteValue := convert.ByteToStringArray(dnsValue)
```

Comment:
You use websiteValue a lot. You should clarify what your are manipulating using a clearer functin name
```suggestion
	dnsRecords := convert.ByteToStringArray(dnsValue)
```